### PR TITLE
RSS-ECOMM-1 development environment configuration and scripts

### PR DESCRIPTION
**Pull request by SvetaIlina**
## Overview
Add Environment Configuration and Scripts

**What has been done:**
- Set up and configure Vite, ESLint, Prettier, Husky
- Configure TypeScript
- Install and configure Jest

**Issue(s) closed:**
- Closes RSS-ECOMM-1_08-RSS-ECOMM-1_16 - Development Environment Configuration/Development Scripts 

## Type of Change
- [x] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

### CODE REVIEW

File: kickFlip-project/package.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+    "name": "kickflip-project",
+    "private": true,
+    "version": "0.0.0",
+    "type": "module",
+    "scripts": {
```
Comment (makarovaiuliia) [inline kickFlip-project/package.json:6]: Привет! А есть скрипт для коммита для хаски?

**Comment by makarovaiuliia** [comment]
Добавь, пожалуйста, скрипты в readme файл 🙏

File: kickFlip-project/package.json (hunk 1 above)
Comment (SvetaIlina) [reply kickFlip-project/package.json:6]: привет! скрипт для комита добавила. в хаски на пре-комит стоит проверка притера, на пре-пуш - линтер.

Comment (Pavel-Kuvshinov) [reply kickFlip-project/package.json:6]: All is fine

**Review by Pavel-Kuvshinov — APPROVED** [review]
All is fine.

**Pull request merged**